container-conf/rsentry.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+# Non-root users get the full {build_run_user_home} environment in the container.
+if (( $EUID != 0 )); then
+    # No core files. If someone wants a corefile, the can raise the limit.
+    ulimit -c 0
+    unset PYTHONPATH
+    unset PYTHONSTARTUP
+    export HOME={build_run_user_home}
```

Comment:
Should this be `export HOME=${build_run_user_home}` (add $)?

Reply:
This is passing through [build_replace_vars](https://github.com/radiasoft/containers/blob/b6b78588168334f5cd7dba0be6243478e79de3be/bin/build.sh#L337), so no quotes.

Reply:
Ah, thanks.